ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -50,7 +69,7 @@ mkdir /opt/rocm-perftest
 pushd ~/rdma-perftest
 ./autogen.sh
 ./configure --enable-rocm --with-rocm=/opt/rocm --prefix=/opt/rocm-perftest/
-make -j 32
+make -j
```

Comment:
should this also not be `make -j$(nproc)`?

Reply:
done